platforms/software/dependency-management/src/main/java/org/gradle/internal/component/model/DependencyMetadata.java

```diff
@@ -28,6 +29,8 @@
 /**
  * A dependency that can participate in dependency resolution.
  * Note that various subtypes provide additional details, but these are not required by the core resolution engine.
+ *
+ * @implSpec See the specification note on {@link #selectVariants(GraphVariantSelector, ImmutableAttributes, ComponentGraphResolveState, AttributesSchemaInternal, Collection)}
```

Comment:
💭 This class-level `implSpec` seems a bit extra to me and I don't find that it really adds any value